geoportal/c2cgeoportal_geoportal/views/tinyowsproxy.py

```diff
@@ -135,13 +135,13 @@ def _get_headers(self):
             headers["Host"] = self.settings.get("tinyows_host")
         return headers
 
-    def _proxy_callback(self, operation, role_id, cache_control, *args, **kwargs):
+    def _proxy_callback(self, operation, role: static.Role, cache_control, *args, **kwargs):
```

Comment:
Shouldn't be user instead of role here ?

Reply:
Yes, thanks :-)